src/pages/admin/create-notes.tsx

```diff
@@ -60,24 +48,10 @@ interface State {
   title: string
 }
 
-class AuthIndexApp extends React.Component<Props, State> {
-
-  render() {
-      if (this.props.authState === "signedIn") {
-          return (
-              <div>
-                  <IndexApp></IndexApp>
-              </div>
-          );
-      } else {
-          return null;
-      }
-  }
-}
-
-class IndexApp extends React.Component<Props, State> {
+// eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
I did some more reading this morning and figured it out thanks to a bug filed about this rule. Here's the correct type
```
type WithoutKeys<T> = Omit<T, keyof T>;

export type EmptyProps = WithoutKeys<Record<string, never>>;
```
I'd put that in a common `types.ts` or `utils.ts` file to use it whenever we need to.

Reply:
Huh interesting. Thanks Nik!

Could you link me to the bug you mentioned?

Reply:
https://github.com/typescript-eslint/typescript-eslint/issues/2063